lib/sorcery/providers/github.rb

```diff
@@ -26,7 +26,7 @@ def get_user_hash(access_token)
 
         auth_hash(access_token).tap do |h|
           h[:user_info] = JSON.parse(response.body).tap do |uih|
-            uih['email'] = primary_email(access_token) if scope =~ /user/
+            uih['email'] = primary_email(access_token) if scope&.include?('user')
```

Comment:
I'm not sure if this is functionally equivalent (moving from a regex match to `.include?` - in particular, how regex matching handles case sensitivity.